spec/controllers/books_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
   include_context 'with JSON API'
 
   # While automatically inferred, they can be also manually specified:
-  base_path '/topics/{topic_id}/books'
+  base_path '/topics/:topic_id'
```

Comment:
This is not proper setup, [swagger requires path variables to be written as curly brackets](https://swagger.io/docs/specification/describing-parameters/#path-parameters)

The postman exporter actually transforms the `{variable}` style into the `:variable` style by itself.

Reply:
Oh, I see where I went wrong, thanks!
Changes made in latest commit.